src/main/java/com/snowflake/kafka/connector/internal/SnowflakeErrors.java

```diff
@@ -340,7 +340,11 @@ public enum SnowflakeErrors {
       "Could not allocate thread for file cleaner to start processing in given time. If problem"
           + " persists, please try setting snowflake.snowpipe.use_new_cleaner to false"),
   ERROR_5025(
-      "5025", "Unexpected data type", "Unexpected data type encountered during schema evolution.");
+      "5025", "Unexpected data type", "Unexpected data type encountered during schema evolution."),
+  ERROR_5026(
```

Comment:
My understanding of SnowflakeErrors is that they represent errors exposed to the customer. In this particular situation we use exception to handle one of the expected scenarios (let's hope uncle Bob is not around) so I would choose an ordinary package-private exception. Wdyt?

Reply:
well, the exception ends up in dlq so it's kinda public,  i would leave it as it is so the customer can find the root cause faster by the error code?

Reply:
Didn't know that it's visible. Let's leave it as it is.